### ISSUE TITLE
build failures with `no entry found for key` panics

### ISSUE THREAD

**Issue by scottlamb** [rust-lang/docs.rs#1632]
### Crate names

- [x] off-rs 0.1.3
- [x] retina 0.3.7
- [x] rp2040-pac 0.3.0
- [x] num-prime 0.1.0
- [ ] holochain 0.0.124
- [x] kube 0.68.0
- [x] themelio-stf 0.9.7

### Build failure links

* https://docs.rs/crate/retina/0.3.7/builds/505117
* https://docs.rs/crate/holochain/0.0.124/builds/505120

### Additional details

Looks like there's been a recent regression in nightly rustdoc that affects more than just my crate. These look related:
* https://github.com/rust-lang/rust/issues/93428
* https://github.com/rust-lang/rust/pull/88679#issuecomment-1023907574
* https://github.com/rust-lang/rust/issues/93476
* https://github.com/rust-lang/rust/issues/93698

Is it possible/worthwhile to pin `docs.rs` to an older nightly until this is fixed?

**Comment by Nemo157** [comment]
There is not a noticeably higher percentage of build failures over the last day or two, so it seems like it's not a very common failure. There's a pretty small fix PR already posted, so we can queue rebuilds for the crates impacted in a couple days when that is released to nightly.

**Comment by Nemo157** [comment]
(Speaking of, it'd be useful to have some scripting to grep recent build logs so we can find all impacted crates/versions).

**Comment by cmpute** [comment]
I guess I'm affected by the same problem: https://docs.rs/crate/num-prime/0.1.0/builds/501949

**Comment by kpreid** [comment]
The original issue was marked fixed, but there's still a panic with `rustdoc 1.60.0-nightly (08df8b81d 2022-01-30)` and some discussion of it on maybe-duplicate [#93476](https://github.com/rust-lang/rust/issues/93476)

**Comment by Nemo157** [comment]
So it sounds like there are multiple sources of this ICE, just in case the specific issues these crates hit are fixed I've requeued builds for `retina 0.3.7`, `off-rs 0.1.3`, `rp2040-pac 0.3.0` and `num-prime 0.1.0`. Then once 93476 is closed we can try again.

**Comment by Nemo157** [comment]
Only `off-rs` appears to have succeeded. I've put details for all the crates hitting this and the new rust issue in the OP.

**Comment by thejpster** [comment]
Hopefully the upstream bug is fixed now?

**Comment by Nemo157** [comment]
Requeued the crates that have been mentioned (`retina`, `rp2040-pac`, `num-prime`, `holochain`, `kube`). If they all succeed then we might requeue all the other crates that have hit this issue too.

**Comment by Nemo157** [comment]
`rp2040-pac`, `num-prime` and `kube` appear to have all worked; `retina` and `holochain` failed again and I've added their new build logs to the OP. I'll open a new upstream issue and wait on requeuing unasked for crates (if you run into/have published a crate that hit this issue, feel free to ask for a rebuild).

**Comment by syphar** [comment]
@Nemo157 there is a new failure reported (by @TheDarkula ) 
https://docs.rs/crate/themelio-stf/0.9.7/builds/502388

is it worth requeuing?

**Comment by Nemo157** [comment]
Yep, that build was before the last fix so might succeed, I've requeued it.

**Comment by TheDarkula** [comment]
@syphar @Nemo157 Thank you!
Is there somewhere I can view the queue?

**Comment by syphar** [comment]
@TheDarkula can find the queue [here](https://docs.rs/releases/queue), rebuilds are lowest prio. 

Looking at https://docs.rs/crate/themelio-stf/0.9.7/builds 

it actually looks like the build passed now.

**Comment by TheDarkula** [comment]
Fantastic!
Thank you so much :)

**Comment by Nemo157** [comment]
Now that we have a new fix I have requeued retina and holochain, and they've both already completed successfully. Since there's no more specific known failures I've also requeued the other 95 matching build failures at a lower priority (also so we can see whether any of them fail again to give another testcase).

<details><summary>other queued crates</summary>

```
abe_gpsw 0.3.0
all-is-cubes 0.4.0
all-is-cubes-content 0.4.0
all-is-cubes-gpu 0.4.0
AmanitaMuscaria 0.0.4
arti-client 0.0.4
arti-config 0.0.4
asn1-codecs 0.1.3
asn1-rs 0.3.0
bevy_rapier2d 0.12.1
bevy_rapier3d 0.12.1
bls-signatures 0.11.3
blstrs 0.4.2
cargo-raze 0.14.2
chia 0.1.0
clvmr 0.1.20
consalifold 0.1.8
consalifold 0.1.9
consprob 0.1.9
cv-convert 0.17.2
d7sneakers 0.3.1
deno_runtime 0.44.0
deno_webstorage 0.31.0
der-parser 6.0.1
dolby_vision 1.6.2
dtn7 0.17.3
filecoin-hashers 6.0.1
filecoin-proofs 11.0.1
fr32 4.0.1
genome-graph 0.6.2
hls 0.0.9
holochain 0.0.126
holochain_cascade 0.0.25
holochain_cascade 0.0.26
holochain_cli 0.0.26
holochain_cli 0.0.27
holochain_cli_bundle 0.0.21
holochain_cli_bundle 0.0.22
holochain_cli_sandbox 0.0.22
holochain_cli_sandbox 0.0.23
holochain_conductor_api 0.0.25
holochain_conductor_api 0.0.26
holochain_keystore 0.0.25
holochain_p2p 0.0.25
holochain_p2p 0.0.26
holochain_sqlite 0.0.25
holochain_sqlite 0.0.26
holochain_state 0.0.25
holochain_state 0.0.26
holochain_types 0.0.25
holochain_types 0.0.26
holochain_wasm_test_utils 0.0.25
holochain_wasm_test_utils 0.0.26
holochain_zome_types 0.0.23
httpflv 0.0.6
libremarkable 0.6.0
lt-fm-index 0.5.0
melwallet-client 0.4.1
menmos 0.2.3
menmos-client 0.1.1
menmos-interface 0.1.1
menmos-protocol 0.1.1
miio-proto 0.1.4
mil 0.2.5
nrf51-pac 0.11.0
nrf52805-pac 0.11.0
nrf52810-pac 0.11.0
nrf52811-pac 0.11.0
nrf52820-pac 0.11.0
nrf52832-pac 0.11.0
nrf52833-pac 0.11.0
nrf52840-pac 0.11.0
nrf5340-app-pac 0.11.0
nrf5340-net-pac 0.11.0
nrf9160-pac 0.11.0
nvd_cve 0.1.0
oberon 1.1.1
reef 0.1.16
rna-algos 0.1.28
rtmp 0.0.12
rustybam 0.1.28
signature_bbs_plus 0.35.0
signature_bls 0.33.0
signature_ps 0.33.0
sqlite2parquet 0.3.0
storage-proofs-core 11.0.1
storage-proofs-porep 11.0.1
storage-proofs-post 11.0.1
storage-proofs-update 11.0.1
subxt-codegen 0.17.0
subxt-macro 0.17.0
themelio-nodeprot 0.10.10
tor-dirmgr 0.0.4
varlociraptor 4.11.2
xiu 0.0.10
```

</details>
